liquidprompt

```diff
@@ -772,6 +775,42 @@ _lp_color_map() {
     ret="${LP_COLORMAP[_LP_FIRST_INDEX+value*${#LP_COLORMAP[*]}/scale]}"
 }
 
+# Generate a terminal hyperlink from a URL and text.
+_lp_create_link() {  # url, text
+    (( LP_ENABLE_HYPERLINKS )) || return 2
+
+    local ret
+    __lp_escape "$1"
+
+    link="$_LP_OPEN_ESC"$'\E]8;;'"${1}"$'\E'"${_LP_BACKSLASH}${_LP_CLOSE_ESC}${2}${_LP_OPEN_ESC}"$'\E]8;;\E'"${_LP_BACKSLASH}$_LP_CLOSE_ESC"
```

Comment:
Something is off here. Why run a string through `__lp_escape()` and then not use the return value?

Either `$1` or `$2` should be escaped, or maybe both... Or neither? We need to track down where the escaping is suppose to happen.

Reply:
Probably neither of them should be escaped? If I want to add formatting in the text, I for sure don't want it to be escaped. And if the URL is escaped, it has the potential to break something. I will go for no escape at all.